_inc/client/components/product-expiration/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { moment, translate as __ } from 'i18n-calypso';
+
+class ProductExpiration extends React.PureComponent {
+	static propTypes = {
+		expiryDate: PropTypes.string.isRequired,
+		purchaseDate: PropTypes.string,
+		isRefundable: PropTypes.bool,
+		dateFormat: PropTypes.string,
+		fallback: PropTypes.string,
+	};
+
+	static defaultProps = {
+		purchaseDate: '',
+		isRefundable: false,
+		dateFormat: 'LL',
+		fallback: '',
+	};
+
+	render() {
+		const { expiryDate, purchaseDate, isRefundable, dateFormat, fallback } = this.props;
+
+		// Return nothing if we don't have any dates.
+		if ( ! expiryDate && ! purchaseDate ) {
+			return fallback;
```

Comment:
I think this `fallback` is unnecessary overhead. I'd very much prefer to return `null` here and handle this with when using this component with something like `{<ProductExpiration /> || 'something'}`. It would also be more flexible.

Reply:
Fixed in 84727755f71988299f54bf4e06f3dcc99d6171ab